examples/conditional-exit/example.py

```diff
@@ -21,7 +23,12 @@ def someone_says_the_magic_word(events):
         return True
 
     # This will run only until someone enters the magic word(s) into a channel.
-    app.run(until=someone_says_the_magic_word)
+    try:
+      app.run(until=someone_says_the_magic_word)
+    except MissingToken:
+      sys.exit('Unable to find Slack API token. \nPlease make sure you have '\
```

Comment:
You shouldn't have to use a `\` here. `sys.exit()` should receive the whole string even without that character.